packages/react/src/components/Dropdown/DropdownSearchInput.tsx

```diff
@@ -122,8 +124,8 @@ class DropdownSearchInput extends UIComponent<ReactProps<DropdownSearchInputProp
           onBlur: this.handleInputBlur,
           onKeyDown: this.handleInputKeyDown,
           ...accessibilityInputProps,
+          ...unhandledProps.input,
```

Comment:
Looks weird, but I understand _why_ it's done.

Reply:
yea..no better idea at this point

Reply:
share the same feeling as @layershifter expressed 😕